api/Document.json

```
@@ -1733,29 +1733,38 @@
             "chrome_android": {
               "version_added": "18"
             },
-            "edge": {
-              "version_added": "12"
-            },
+            "edge": [
+              {
+                "version_added": "17"
+              },
+              {
+                "version_added": "12",
+                "partial_implementation": true,
+                "notes": "Before Edge 17, the <code>oncopy</code> attribute was not supported. To listen to this event, use <code>document.addEventListener('copy', function() {});</code>."
```

Per our guidelines, we have a standard sentence for this. https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#dom-events-eventname_event
```suggestion
                "notes": "The <code>oncopy</code> event handler property is not supported."
```